metacat-main/src/main/java/com/netflix/metacat/main/services/impl/PartitionServiceImpl.java

```diff
@@ -365,7 +370,7 @@ public void delete(final QualifiedName name, final List<String> partitionIds) {
                 deleteMetadatas(metacatRequestContext.getUserName(), partitions);
             }
             eventBus.postAsync(
-                new MetacatDeleteTablePartitionPostEvent(name, metacatRequestContext, this, partitionIds)
+                new MetacatDeleteTablePartitionPostEvent(name, metacatRequestContext, this, partitionDtos)
```

Comment:
On delete we can just publish the partition ids. DO we need to publish the whole partition objects?